### PR TITLE
Added blog post "Newbie in RDO (2): RDO Trunk from a bird's eye view"

### PR DESCRIPTION
**Pull request by amoralej**
This is a new post with a high level overview of RDO trunk. As for the previous post, I've used the new repos model based on buildlogs so we shouldn't publish until these repos are ready unless we prefer to publish it with the current model. 

Feedback is apreciated.

### CODE REVIEW

**Comment by amoralej** [comment]
Although the table doesn't look fine in github is properly redered in the server.

File: source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+NVR for packages generated by DLRN follows some rules:
+
+- **Version** is set to MAJOR.MINOR.PATCH of the *next* upstream version.
+- **Release** is 0.&lt;timestamp>.&lt;short commit hash>
+
+For example, python-novaclient-3.4.1-0.20160412103538.77f214b.el7.centos.noarch.rpm is a package containing code released after GA of 3.4.0 and before 3.4.1 containing code until upstream commit [77f214b](https://github.com/openstack/python-novaclient/commit/77f214bdbd7d9e8ba935ace1dce0290061ddd737).
+
+Once the package has been successfully built, DLRN creates a yum repository containing the latest version of every package successfully built at that time. This repo will not change and is created in a unique directory based on the specific commit hash. These repos will be available if a user want to test an installation based on a specific commit. Finally, DLRN manages two "shorcuts" for repositories:
```
Comment (javierpena) [inline source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md]: ... if a user wants ...

File: source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+- **current**: will point to the last created per-commit repo.
+- **consistent**: current repo, may not contain the latest versions of packages that failed to build (i.e. let's assume that packaging of python-nova package failed while a later build of python-cinder succeed, in this case, current would contain latest upstream version of cinder but not for nova). To avoid this situation, a new pointer "consistent" is managed by DLRN and will point to the last snapshot that contained packages versions in sync with upstream version.
```
Comment (javierpena) [inline source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md]: ... that contained package versions...

File: source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+Also, DLRN creates some html reports that can be used to check the results of each single build and provides the logs to troubleshoot any error while building the package.
+
+## Continuous Integration in RDO Trunk
+
+To ensure the packages built by DLRN work as expected, the RDO project manages some CI jobs pipeline in the [CentOS CI](https://ci.centos.org/view/rdo/) system. These jobs are managed using jenkins-job-builder with the configuration manated in [rdo-infra github](https://github.com/rdo-infra/ci-config). These jobs deploy and test RDO in different scenarios (packstack, tripleo quickstart and puppet) based mostly on weirdo (take a look into [weirdo documentation](http://weirdo.readthedocs.org/en/latest/how.html)) using the **consistent** repository. If all tests in the pipeline succes, the content of repo will be promoted as **current-passed-ci**. In this way, this link always points to the latest consistent repo that passed the CI jobs.
```
Comment (javierpena) [inline source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md]: s/weirdo/WeIRDO/

File: source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+Additionally, **current-tripleo** repo is managed by a set of jobs focused on testing TripleO installer on the current-passed-ci repo. When those tests pass, this repo is promoted by this CI pipeline.
+
+## Access to repositories
+
+All the repositories and content managed by DLRN and CI jobs are not directly accessible for public usage from DLRN server. Instead, the content s replicated to a different destinations depending on the type:
```
Comment (javierpena) [inline source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md]: ... the content is replicated to different destinations...

File: source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+|centos7-mitaka/consistent | |  http://buildlogs.centos.org/centos/7/cloud/x86_64/rdo-trunk-mitaka
+|centos7-mitaka/current-passed-ci | |  http://buildlogs.centos.org/centos/7/cloud/x86_64/rdo-trunk-mitaka-tested
+|centos7-liberty/consistent | |  http://buildlogs.centos.org/centos/7/cloud/x86_64/rdo-trunk-liberty
+|centos7-liberty/current-passed-ci | |  http://buildlogs.centos.org/centos/7/cloud/x86_64/rdo-trunk-liberty-tested
+|centos7-kilo/consistent | |  http://buildlogs.centos.org/centos/7/cloud/x86_64/rdo-trunk-kilo
+|centos7-kilo/current-passed-ci | |  http://buildlogs.centos.org/centos/7/cloud/x86_64/rdo-trunk-kilo-tested
+
+<br>
+
+- By-commit repos (the ones based on the commit hash), build reports and logs are synchronized to the build server and are accesible in [https://trunk.rdoproject.org/](https://trunk.rdoproject.org/) to developers or packages. On each repository directory a delorean.repo file is created containing the yum configuration required to enable it easily.
```
Comment (javierpena) [inline source/blog/2016-04-12-newbie-in-rdo-2-rdo-trunk-from-a-bird-s-eye-view.html.md]: s/accesible/accessible/g

... to developers or packagers.

**Pull request merged**